bootstrap-salt.sh

```diff
@@ -2394,9 +2394,9 @@ __create_virtualenv() {
 __activate_virtualenv() {
     set +o nounset
     # Is virtualenv empty
-    if [ -z "$VIRTUAL_ENV" ]; then
+    if [ -z "$_VIRTUALENV_DIR" ]; then
         __create_virtualenv || return 1
-        # shellcheck source=/dev/null
+        # shellcheck disable=34
```

Comment:
I think this is rule [SC1090](https://github.com/koalaman/shellcheck/wiki/SC1090).
What is `34` by the way? I have tried that with ShellCheck 0.4.7 and it does not work.